addons/website_event_track_quiz/static/src/scss/event_quiz.scss

```diff
@@ -24,7 +24,7 @@ $o-wslides-fs-side-width: 300px;
         top: -40px;
 
         @include media-breakpoint-up(lg) {
-            font-size: 1rem;
+            @include font-size(1rem);
```

Comment:
Maybe not supper useful as the rule is already in a media query...

Reply:
Yes, when it is in a media query, I would check if the CSS can be simplified or not

Reply:
To check !

Reply:
Maybe I'm missing something, but I have the impression that some of the scss rules are no longer in use here :thinking: ... I don't see how we can have a `o_wslides_home_nav`  in a `o_quiz_main`. I'll leave the font size statement as it was, but maybe we should report this (probably dead code) to the SM team.

Reply:
Indeed, let's not touch that. And indeed, if you can report it that's cool :+1:

Reply:
Reported in discord: https://discord.com/channels/678381219515465750/775727120193814528/1143217190754070719